### DISCUSSION TITLE
.NET MAUI 8.0.60 (Service Release 6)

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#23007] [Announcements]
This release is now available via [NuGet.org](https://www.nuget.org/packages/Microsoft.Maui.Controls/8.0.60) and is compatible with the latest .NET 8 workload. 

```
<ItemGroup>
  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.60" />
  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.60" />
</ItemGroup>
```

The focus of our service releases is resolving high-impact bugs for developers upgrading their applications from Xamarin to .NET 8.

- [Release notes](https://github.com/dotnet/maui/releases/tag/8.0.60)

**Comment by Phantom-KNA** [comment]
Anything new about collectionview view??
